src/store/project/tests/page-ref-test.ts

```diff
@@ -0,0 +1,101 @@
+// tslint:disable:no-object-literal-type-assertion
+
+import { PageRef } from '../page-ref';
+import { Project } from '../project';
+import { Store } from '../../store';
+
+describe('Store PageRef', () => {
+	const testData = {
```

Comment:
I'd prefer to create the required test data in a small utility function local to the test file instead of defining it in the `describe` closuer